src/store/actionCreators/saveSets.ts

```diff
@@ -72,6 +81,36 @@ export const createSaveSetIfUnique = (
   }
 };
 
+export const editSaveSet = (
+  payload: EditSetParams,
+): ThunkAction<void, RootState, null, SaveSetsActionTypes> => async (dispatch) => {
+  const { saveSetInfo, onNameConflict, onSuccess, onFail } = payload;
+
+  try {
+    const tagIsUnique = (await saveSetCountForTag(saveSetInfo.name, saveSetInfo.currentUser)) === 0;
+    if (tagIsUnique) {
+      const result = await editSaveSetTag(saveSetInfo);
```

Comment:
result is a bit vague for me in the sense that I don t know what I should expect for result. Maybe you could use a more meaningful name for instance nOfSaveSetEdited or something in that sense.